### PR TITLE
bitnami/rabbitmq: add support for ingressClassName

### PR DESCRIPTION
**Pull request by jiphex**
**Description of the change**

Add support for the `ingressClassName` field to the `rabbitmq/` chart.

**Benefits**

Chart users will be able to make use of the RabbitMQ chart with a non-default Ingress

**Possible drawbacks**

None.

**Additional information**

Implementation copied from the wordpress chart

**Checklist**
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

**Review by miguelaeh — APPROVED** [review]
Thank you very much for the PR!
LGTM!

**Comment by miguelaeh** [comment]
Hi @jiphex ,
Please bump the chart version to be able to merge the PR

**Comment by jiphex** [comment]
Done 👍

File: bitnami/rabbitmq/Chart.yaml [hunk 1]
```diff
@@ -23,4 +23,4 @@ name: rabbitmq
 sources:
   - https://github.com/bitnami/bitnami-docker-rabbitmq
   - https://www.rabbitmq.com
-version: 8.16.3
+version: 8.16.4
```
Comment (miguelaeh) [inline bitnami/rabbitmq/Chart.yaml:26]: I am sorry, since you are adding a new parameter this should be a minor bump, not a patch

**Review by miguelaeh — CHANGES_REQUESTED** [review]

File: bitnami/rabbitmq/Chart.yaml (hunk 1 above)
Comment (jiphex) [reply bitnami/rabbitmq/Chart.yaml]: Done

**Pull request merged**